buildstockbatch/test/test_inputs/test_openstudio_buildstock/resources/buildstock.rb

```diff
@@ -1,8 +1,7 @@
 # frozen_string_literal: true
 class Version
-  ResStock_Version = '3.1.0' # Version of ResStock
+  ResStock_Version = '2000.1.0' # Version of ResStock
```

Comment:
Test against the new `WorkflowGenerator_Version` line?

Reply:
Hmm, I forget why I made this change. ResStock_Version doesn't really matter for BuildStockBatch. We only care about "WorkflowGenerator_Version". I will address this.